CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 ## 1.24.0 (Unreleased)
 
+
+## 1.23.11 (May 19, 2021)
+
+* **New Resource:** `rds_replica_v3` ([#1037](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1037))
+
+ENHANCEMENTS:
+* `resource/opentelekomcloud_lb_pool_v2`: Make `persistence.type` `Optional` ([#1052](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1052))
+* `resource/opentelekomcloud_vpc_eip_v1`: Add possibility to query by `tags` ([#1058](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1058))
+
+BUG FIXES:
+* `resource/opentelekomcloud_compute_instance_v2`: Fix issue with reading `networks` ([#1057](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1057))
+* `resource/opentelekomcloud_identity_provider_v3`: Fix issue with clients ([#1059](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1059))
```

Comment:
It's more about failing provider operations. From user POV there is no issue with clients.